modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1141,7 +1142,7 @@ public final class IgniteSystemProperties {
      * index store is disabled.
      */
     @SystemProperty(value = "Maximum payload size in bytes for H2TreeIndex. " +
-        "0 means that inline index store is disabled", type = Integer.class, defaults = "" + IgniteUtils.MAX_INLINE_SIZE)
+        "0 means that inline index store is disabled", type = Integer.class, defaults = "" + IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT)
```

Comment:
I already asked this question but I forgot the answer. Before introducing the change you are reverting it was 10, not 64. Why is it 64 now?